anime_downloader/commands/dl.py

```diff
@@ -79,10 +80,11 @@
     help="Set the speed limit (in KB/s or MB/s) for downloading when using aria2c",
     metavar='<int>K/M'
 )
+@click.option("--video-info", "-i", is_flag=True, help="enables getting video information")
```

Comment:
Capitalise the first letter to match the casing of other help messages. Additionally, add a sentence explain that this requires opencv-python